File: Modul 2/Soal 4/PRAK204-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
#include <math.h>

int main () {
    float jari_jari, tinggi, volume, luas, keliling;
    const float pi = 22.0/7.0;

    printf("Masukkan Jari-jari: ");
    scanf("%f", &jari_jari);
    printf("Masukkan Tinggi: ");
    scanf("%f", &tinggi);

    volume = pi * jari_jari* jari_jari * tinggi;
    luas = 2 * pi * jari_jari * (jari_jari + tinggi);
    keliling = 2 * pi * jari_jari;

    printf("Volume = %.2f \n", volume);
    printf("Luas = %.2f \n", luas);
    printf("Keliling = %.2f \n", keliling);

   return 0; 
}

File: Modul 2/Soal 4/PRAK204-2410817220018-ANGGRAENI DWI ZAHRA.py
import math

jari_jari = float(input ("Masukkan Jari-jari : "))
tinggi = float(input ("Masukkan Tinggi : "))

volume = 22/7 * jari_jari**2 * tinggi
luas = 2 * 22/7 * jari_jari * (jari_jari + tinggi)
keliling = 2 * 22/7 * jari_jari

print(f"Volume =  {volume:.2f}")
print(f"Luas =  {luas:.2f}")
print(f"Keliling = {keliling:.2f}")